recipes/imagecodecs/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2019.11.18" %}
+
+package:
+  name: imagecodecs
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/imagecodecs/imagecodecs-{{ version }}.tar.gz
+  sha256: 037e3fde4592fffdb4b8235170180d0d0ba01e4640d9660360dcc62f062e2bd1
+
+build:
+  number: 0
+  entry_points:
+    - imagecodecs=imagecodecs.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - cython
+    - msinttypes  # [win]
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - imagecodecs
+  commands:
+    - imagecodecs --help
+  requires:
```

Comment:
```suggestion
```

Reply:
This is still a test. I was adding the c-libs one-by-one here.